qemu/tests/virtio_fs_share_data.py

```diff
@@ -211,6 +211,20 @@ def get_viofs_exe(session):
                 if md5_guest != md5_host:
                     test.fail('The md5 value of host is not same to guest.')
 
+                # during virtio fs is mounted, reboot vm
+                if params.get('reboot_guest', 'False'):
```

Comment:
```suggestion
                if params.get('reboot_guest', 'no') == 'yes':
```
`if params.get('reboot_guest', 'False'):` , it will be `True` all the time.